src/runtime/runtime.ts

```diff
@@ -176,7 +177,7 @@ export class Runtime {
   // How best to provide default storage to an arc given whatever we decide?
   newArc(name: string, storageKeyPrefix?: ((arcId: ArcId) => StorageKey), options?: RuntimeArcOptions): Arc {
     const {loader, context} = this;
-    const id = IdGenerator.newSession().newArcId(name);
+    const id = options && options.id ? options.id : IdGenerator.newSession().newArcId(name);
```

Comment:
We can make use of javascript logical operators to make this more succinct, too: 

```
(options && options.id) || IdGenerator.newSession().newArcId(name);
```

I understand why we might reject this suggestion; it might be a bit unclear as to what the line returns.

Reply:
i like this suggestion, done.